README.md

```diff
@@ -46,12 +46,15 @@ To see how to get it working with RedHat/Centos 7 check out these [docs](https:/
 
 We provide [an official Docker image in the GitHub Container Registry](https://github.com/orgs/voxpupuli/packages/container/package/puppetboard).
 
+You must provide a secret key. Generate one by running `ruby -e "require 'securerandom'; puts SecureRandom.hex(32)"`.
```

Comment:
Nit: I would suggest wording like "Generate one for example by running" to prevent assumptions that this command is the only way to generate a secret key.